java-manta-client/src/test/java/com/joyent/manta/client/crypto/AesCbcCipherDetailsTest.java

```diff
@@ -8,86 +8,104 @@
 package com.joyent.manta.client.crypto;
 
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import java.security.NoSuchAlgorithmException;
+
 import javax.crypto.SecretKey;
 
 @Test
 public class AesCbcCipherDetailsTest extends AbstractCipherDetailsTest {
+
+    private AesCbcCipherDetails AES_CBC_128;
+
+    private AesCbcCipherDetails AES_CBC_192;
+
+    private AesCbcCipherDetails AES_CBC_256;
+
+    @BeforeClass
+    private void init() throws NoSuchAlgorithmException {
```

Comment:
The fact that this `init` method was defined on every test that interacts with any of the AesCipherDetails classes proves you're aware of the problems caused by removing the `INSTANCE` constants.